src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,97 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_CAPACITY = 10;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
```
так як не потрібен доступ до змінної ззовні краще зробити private